src/libcore/mem.rs

```diff
@@ -84,7 +86,7 @@ pub use intrinsics::transmute;
 /// mem::forget(file);
 /// ```
 ///
-/// The swap function uses forget to good effect.
+/// The `mem::swap()` function uses forget to good effect:
```

Comment:
I've tended to always use parens when talking about a `function()`. I guess that's not official style...